Pat.Sender/Extensions/BrokeredMessageExtensions.cs

```diff
@@ -37,22 +38,46 @@ public static string GetCorrelationId(this Message message)
             return message.UserProperties["PBCorrelationId"]?.ToString();
         }
 
-        public static long GetSize(this Message message)
+        /// <summary>
+        /// Calculate the estimated message size in bytes.
+        /// Based on: https://weblogs.asp.net/sfeldman/asb-batching-brokered-messages
+        /// </summary>
+        /// <param name="message">The message whose size to calculate.</param>
+        /// <returns>The message size in bytes.</returns>
+        public static long GetEstimatedMessageSize(this Message message)
```

Comment:
Can this extension be made `internal`? Also by renaming the `GetSize` method this is a breaking change. Possibly keep the existing name?

Reply:
I don't object to the new method name as it is accurate, AFAIK these are only used internally so could even make the entire class internal, although then would need to add `[assembly: InternalsVisibleTo("PB.ITOps.Messaging.PatSender.UnitTests")]`